<H2>Description</H2>
<P>
Exon Usage data aggregated across 19 ENCODE cell lines. The green marks represent 5' splice sites. The blue marks represent 3’ splice sites.

<H2>Methods</H2>
<P>
The splice site usage of a 5’ or 3’ splice site is meant to estimate the proportion of transcripts from a gene that undergo a splicing event that utilizes that particular site.
To accomplish this we analyze exon-exon junction reads obtained from mapping of RNA-seq data. For a given splice site, there are three categories of junction reads which go into calculating its splice site usage:
(A) a read that has the splice site as one of the sides of the junction, (B) a read that spans the splice site (i.e. the junction is between a site that is upstream and one that is downstream of the site in question), 
and (C) for a 5’ (3’) splice site, a read with a junction that has its 5’ (3’) end in the downstream (upstream) intron. From the counts of these three classes of reads, splice site usage is defined as A/(A+B+C).
In order to mitigate the corruption of this metric by the false positive splice junctions frequently output by RNA-seq aligners, we only considered junction reads that contained splice sites present in the GENCODEv29 annotation.
If there were multiple RNA-seq replicates for a particular cell line or condition, we collapsed the junction read counts from all replicates before calculating splice site usages.
<P>
In other words, splice site usage measures how often a particular splice site is "used" during splicing relative to other splice sites in the same gene, with values ranging from 0 (never used) to 1 (every transcript seen in the RNA-seq data uses this splice site).
Scripts used to generate usage data can be found <a href="https://github.com/dtglidden/hotspot-exon-paper">here</a>

<P>
Because splice site usage can vary greatly across cell types, the values in this track are the average across the indicated cell lines, limited to those with low (less than 0.001) inter-cell type variance.


<P>
All ENCODE cell lines used in analysis (individual cell line tracks are also provided):
<ul>
<li>A549</li>
<li>AG04450</li>
<li>BJ</li>
<li>CD14</li>
<li>CD20</li>
<li>GM12878</li>
<li>H1-hESC</li>
<li>HeLa-S3</li>
<li>HepG2</li>
<li>HMEC</li>
<li>HSMM</li>
<li>HUVEC</li>
<li>IMR90</li>
<li>K562</li>
<li>MCF-7</li>
<li>NHEK</li>
<li>NHLF</li>
<li>SK-N-SH</li>
<li>SK-N-SH-RA</li>
</ul>

<H2>Credits</H2>
<P>
This track was created at the <a href="http://fairbrother.biomed.brown.edu/index.php">Fairbrother Laboratory</a> at Brown University by Luke Buerer, Camillo Saueressig, and David Glidden.

<H2>References</H2>
<P>
ENCODE Project Consortium. (2012). An integrated encyclopedia of DNA elements in the human genome. Nature, 489(7414), 57.

<H2>Contact</H2>
<P>
william_fairbrother@brown.edu